src/ctelnet.cpp

```diff
@@ -199,6 +199,18 @@ void cTelnet::encodingChanged(const QString& encoding)
         // Do NOT create BOM on out-going text data stream!
         outgoingDataEncoder = outgoingDataCodec->makeEncoder(QTextCodec::IgnoreHeader);
 
+        if (!mEncoding.isEmpty() && mEncoding != QLatin1String("ASCII")) {
+            mpOutOfBandDataIncomingCodec = QTextCodec::codecForName(encoding.toLatin1().constData());
+            qDebug().nospace() << "cTelnet::encodingChanged(" << encoding << ") INFO - Installing a codec for OOB protocols that can handle: " << mpOutOfBandDataIncomingCodec->aliases();
+
+        } else if (mpOutOfBandDataIncomingCodec) {
+            // Will get here if the encoding is ASCII (or empty which is treated
+            // the same) and there is still an an encoder set:
+            qDebug().nospace() << "cTelnet::encodingChanged(" << encoding << ") INFO - Uninstalling the codec for OOB protocols that can handle: " << mpOutOfBandDataIncomingCodec->aliases() << " as the new encoding setting of: "
+                               << encoding << " does not need a dedicated one explicitly set...";
```

Comment:
The ellipsis is unnecessary

Reply:
Really? it is *only* a debug message and that I am thinking that hopefully it is going to be okay to not use an encoder from this point on...

Reply:
Yeah, I'd leave the `...` out, just have a period.